config/routes.rb

```diff
@@ -5,6 +5,11 @@
 
   resources :quizzes do
     resources :questions
+    member do
+      get "take/:question_index", action: :take, as: :take
+      post "submit", action: :submit
+      get "results", action: :results, as: :results
```

Comment:
```suggestion
      post "submit"
      get "results"
```

We don't need the extra arguments for these as they are the defaults